### ISSUE TITLE
phantomjs crashes about 1/3 times (Mac OSX development version)

### ISSUE THREAD

**Issue by DelvarWorld** [ariya/phantomjs#10977]
> 
>     phantomjs --version
>     1.9.0 (development)
> 
> this seems to happen on a page open. it's against a local environment.
> 
> it seems to happen most frequently on loading pages. These pages do contain custom web fonts.
> 
> I have included a sample of the numerous dmp files created.
> 
> The page is has an invalid ssl cert and i'm using ignore-ssl-errors. I don't think that's relevant but trying to include as much info as I can.

**Disclaimer:**
This issue was migrated on 2013-03-15 from the project's former issue tracker on Google Code, [Issue #977](http://code.google.com/p/phantomjs/issues/detail?id=977).
:star2: &nbsp; **3** people had starred this issue at the time of migration.

**Comment by ariya** [comment]
Can you try it again with PhantomJS 2.1?

**Comment by DelvarWorld** [comment]
@ariya that was over two years ago :P I was trying to build a test suite using phantomjs for browser automation. However, in addition to dealing with very complicated async tests, I also had to deal with Phantom crashing almost constantly, meaning my test suite would never finish, and I would never know if my tests passed and my application was bug free, or if phantomjs was crashing. The crashes were frequent and unpredictable, implying the core of phantom was fundamentally unstable. Because this library only caused pain during use, I moved on to headless Firefox (mature, stable), and long ago stopped using Phantomjs. Feel free to close this ticket.
